x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/sourcerer/sourcerer_timeline.cy.ts

```diff
@@ -109,6 +109,7 @@ describe('Timeline scope', { tags: ['@ess', '@serverless', '@brokenInServerless'
     });
 
     it('Modifies timeline to alerts only, and switches to different saved timeline without issue', function () {
+      closeTimeline();
```

Comment:
we need to close the timeline to be able to click on the button to open a new timeline, as it is hidden behind the timeline modal.
This was working previously because there was a `{ force: true }` [here](https://github.com/elastic/kibana/blob/main/x-pack/test/security_solution_cypress/cypress/tasks/timeline.ts#L401) that was removed in this PR [here](https://github.com/elastic/kibana/pull/173886/files#diff-719677c5496f99d1aca519ff457bd922ff4c68e4517de9aa6052355a94ace9f9R401)